src/main/webapp/script.js

```diff
@@ -301,35 +301,81 @@ function createNotificationsElement(notification) {
  * Each element corresponds to a question to be displayed in the DOM.
  */
 function createQuestionElement(question, hasRedirect) {
```

Comment:
Add comments in this function to see what each element is for, and understand what you are actually creating.

Reply:
done